src/types.ts

```diff
@@ -401,7 +431,7 @@ export namespace Sendle {
       /**
        * The unit of measurement for the volume. Most be one of l (litres), m3 (cubic metres), in3 (cubic inches) or ft3 (cubic feet).
        */
-      units: string;
+      units: 'l' | 'm3' | 'in3' | 'ft3' | string;
```

Comment:
I've left these with the `| string` on the end in case someone uses `L` instead of `l`. I'm not sure whether it'd create a breaking change in that case.